src/lib/node/message_handler.ts

```diff
@@ -20,7 +20,13 @@ import { NodeResultTypes } from "./outgoing_message";
 import { dumpNode } from "..";
 
 export class NodeMessageHandler {
-  static async handle(
+  private firmwareUpdateQueued: { [key: number]: boolean };
+
+  constructor() {
+    this.firmwareUpdateQueued = {};
+  }
```

Comment:
These are executed when the class is constructed
```suggestion
  private firmwareUpdateQueued: { [key: number]: boolean } = {};
```